docs/contributing-to-shunter.md

```diff
@@ -1,35 +1,35 @@
-# Guide for Shunter developers
+# Contributing to Shunter
 
-This guide is here to help new developers get started with developing Shunter itself. It will outline the structure of the library and some of the development practices we uphold.
+This guide is here to help new developers get started to contribute with the development of Shunter itself. It will outline the structure of the library and some of the development practices we uphold.
```

Comment:
```suggestion
This guide is here to help new developers get started on contributing to the development of Shunter itself. It will outline the structure of the library and some of the development practices we uphold.
```